pkg/app/app.go

```diff
@@ -90,50 +88,8 @@ func (a *WalletApp) Hide() {
 	runtime.WindowHide(a.Ctx)
 }
 
-type selectFileResult struct {
-	Err         string `json:"err"`
-	CodeMessage string `json:"codeMessage"`
-	FilePath    string `json:"filePath"`
-	Nickname    string `json:"nickname"`
-}
-
-func (a *WalletApp) SelectAccountFile() selectFileResult {
-	filePath, err := runtime.OpenFileDialog(a.Ctx, runtime.OpenDialogOptions{
-		Title:   "Import account file",
-		Filters: []runtime.FileFilter{{DisplayName: "Account File (*.yaml;*.yml)", Pattern: "*.yaml;*.yml"}},
-	})
-	if err != nil {
-		fmt.Printf("error while selecting a file: %v", err)
-		return selectFileResult{Err: err.Error(), CodeMessage: utils.ErrAccountFile}
-	}
-
-	if filePath == "" {
-		return selectFileResult{Err: utils.ErrNoFile, CodeMessage: utils.ErrNoFile}
-	}
-
-	wallet, loadErr := wallet.LoadFile(filePath)
-	if loadErr != nil {
-		fmt.Printf("error while loading file: %v", loadErr.Err)
-		return selectFileResult{Err: loadErr.Err.Error(), CodeMessage: loadErr.CodeErr}
```

Comment:
why deleting all theses methods ?? they are binded function that are called by the frontend

Reply:
you're right